### PR TITLE
Build `nodejs18` images

### PR DESCRIPTION
**Pull request by omBratteng**

### CODE REVIEW

File: cloudbuild_cosign.sh [hunk 1]
```diff
@@ -36,10 +36,13 @@ for distro_suffix in "" -debian11; do
 done
 
 for distro_suffix in "" -debian11; do
-  cosign sign "$@" gcr.io/$PROJECT_ID/nodejs${distro_suffix}:latest
-  cosign sign "$@" gcr.io/$PROJECT_ID/nodejs${distro_suffix}:debug
   cosign sign "$@" gcr.io/$PROJECT_ID/nodejs${distro_suffix}:16
```
Comment (loosebazooka) [inline cloudbuild_cosign.sh:39]: can you move these 16's below the 18's

File: cloudbuild_docker.sh [hunk 2]
```diff
@@ -59,8 +59,11 @@ for java_version in -base 11 17; do
 done
 
 for distro_suffix in "" -debian11; do
-  docker_manifest gcr.io/$PROJECT_ID/nodejs${distro_suffix}:latest "amd64 arm64"
-  docker_manifest gcr.io/$PROJECT_ID/nodejs${distro_suffix}:debug "amd64 arm64"
   docker_manifest gcr.io/$PROJECT_ID/nodejs${distro_suffix}:16 "amd64 arm64"
```
Comment (loosebazooka) [inline cloudbuild_docker.sh:62]: order, same here

File: BUILD [hunk 3]
```diff
@@ -124,20 +124,20 @@ NODEJS = {
     "{REGISTRY}/{PROJECT_ID}/nodejs:14": "//nodejs:nodejs14_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:12-debug": "//nodejs:nodejs12_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:latest": "//nodejs:nodejs16_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:debug": "//nodejs:nodejs16_debug_amd64_debian11",
+    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:latest": "//nodejs:nodejs18_amd64_debian11",
+    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:debug": "//nodejs:nodejs18_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:12": "//nodejs:nodejs12_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:14": "//nodejs:nodejs14_amd64_debian11",
+    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:16": "//nodejs:nodejs16_amd64_debian11",
```
Comment (loosebazooka) [inline BUILD]: I don't think these 16 images should not be added as they are covered by the multi-arch image builds in the cloudbuild

**Comment by loosebazooka** [comment]
so 18 is considered the current active node release?

File: cloudbuild_docker.sh (hunk 2 above)
Comment (omBratteng) [reply cloudbuild_docker.sh]: Done, it's now `latest`, `18` and `16`

File: cloudbuild_cosign.sh (hunk 1 above)
Comment (omBratteng) [reply cloudbuild_cosign.sh]: Done, it's now `latest`, `18` and `16`

File: BUILD (hunk 3 above)
Comment (omBratteng) [reply BUILD]: Yeah, I noticed. I removed `latest` and `16` from there, it gets covered by the manifests

**Comment by omBratteng** [comment]
Yeah, node18 is considered the current node release. It enters LTS schedule in October
image

File: BUILD [hunk 4]
```diff
@@ -122,8 +122,6 @@ PYTHON3.update({
 NODEJS = {
     "{REGISTRY}/{PROJECT_ID}/nodejs:14": "//nodejs:nodejs14_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:latest": "//nodejs:nodejs16_amd64_debian11",
```
Comment (loosebazooka) [inline BUILD:125]: oh interesting, I guess the old config was double pushing these tags... not a real bad issue, but whoops.

Comment (omBratteng) [reply BUILD:125]: That's why I added 18 there, but wasn't exactly sure. I think the manifests overwrote this

**Review by loosebazooka — APPROVED** [review]

**Comment by omBratteng** [comment]
Awesome @loosebazooka, I see `nodejs:18` is now available as an image, and the manifest looks to be working :smile:

```json
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 918,
         "digest": "sha256:a6b4c4841e31e26aeebb4c43c4f0e8f035010843d85f4a5ffafbb1e29ebbad1f",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 918,
         "digest": "sha256:1c8d9943a711063c244ab74e18b1fdd1ffbe60a51553be90d72f5a247360c766",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```

**Pull request merged**